docs/developer-docs/integrations/sns/testing/testing-locally.md

```diff
@@ -5,13 +5,14 @@ sidebar_position: 2
 
 ## Overview
 
-To help developers, DFINITY has created the `sns-testing` repo which has scripts that help developers test the SNS process. Developers can run a local version of the Internet Computer on their local machine, deploy their dapp locally and run through [the stages](../launching/launch-summary.md) of decentralizing their dapp. 
+To help developers, DFINITY has created the [sns-testing repo](https://github.com/dfinity/sns-testing) which has scripts that help developers test the SNS process. Developers can run a local version of the Internet Computer on their local machine, deploy their dapp locally and run through [the stages](../launching/launch-summary.md) of decentralizing their dapp. 
```

Comment:
Should we already say here that this might not work for any dapp?
The repo is just supposed to show this on one example dapp and potentially you can do something similar with your own dapp, but as soon as you need something more customized it would probably be good to following the new guildelines here

Reply:
done